java/registry/src/main/resources/public/swagger-ui.html

```diff
@@ -37,7 +37,7 @@
     window.onload = function() {
         // Begin Swagger UI call region
         const ui = SwaggerUIBundle({
-            url: "/api/docs/swagger.json",
+            url: window.location.pathname.split("/").slice(0, -1).join("/") + "/api/docs/swagger.json",
```

Comment:
Can you make the change to support for relative URL instead of absolute.

Reply:
The change is to make it a relative url only, to get the context url `window.location.pathname.split("/").slice(0, -1).join("/")` is added.
i.e. if swagger-ui endpoint is `https://api.abc.com/registry/swagger-ui.html `then it adds the` /registry` before the api json endpoint